Formula/kyoto-tycoon.rb

```diff
@@ -24,18 +35,29 @@ def install
     system "make"
     system "make", "install"
   end
-end
 
+  test do
+    (testpath/"test.lua").write <<~EOS
+      kt = __kyototycoon__
+      db = kt.db
+      -- echo back the input data as the output data
+      function echo(inmap, outmap)
+         for key, value in pairs(inmap) do
+            outmap[key] = value
+         end
+         return kt.RVSUCCESS
+      end
+    EOS
+    port = free_port
 
-__END__
---- a/ktdbext.h  2013-11-08 09:34:53.000000000 -0500
-+++ b/ktdbext.h  2013-11-08 09:35:00.000000000 -0500
-@@ -271,7 +271,7 @@
-       if (!logf("prepare", "started to open temporary databases under %s", tmppath.c_str()))
-         err = true;
-       stime = kc::time();
--      uint32_t pid = getpid() & kc::UINT16MAX;
-+      uint32_t pid = kc::getpid() & kc::UINT16MAX;
-       uint32_t tid = kc::Thread::hash() & kc::UINT16MAX;
-       uint32_t ts = kc::time() * 1000;
-       for (size_t i = 0; i < dbnum_; i++) {
+    pid = fork do
+      exec bin/"ktserver", "-port", port.to_s, "-scr", testpath/"test.lua"
+    end
+    sleep 5
+
+    assert_match "Homebrew\tCool", shell_output("#{bin}/ktremotemgr script -port #{port} echo Homebrew Cool 2>&1")
+  ensure
+    Process.kill("TERM", pid)
+    Process.wait(pid)
+  end
```

Comment:
This should be handled by Homebrew.

Reply:
Just to confirm: Homebrew auto-kills all `fork`ed processes, so I can remove the `ensure` block?

Reply:
It _should_ kill all forked processes.